clang/test/CXX/temp/temp.decls/temp.variadic/sizeofpack.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -std=c++11 -fsyntax-only -verify %s
+// RUN: %clang_cc1 -std=c++20 -fsyntax-only -verify %s
```

Comment:
We should add - rather than modify - a runline.
The new tests can be protected with `#if __cplusplus >= 202002`